index.d.ts

```diff
@@ -24,6 +24,7 @@ declare namespace nock {
 
   export interface Options {
     allowUnmocked?: boolean;
+    reqheaders:Object
```

Comment:
Yes, thanks. I'd also recommend not using the `Object` type. It doesn't really mean anything, and probably doesn't do what you want. For instance, http://www.typescriptlang.org/play/#src=const%20test%3A%20Object%20%3D%20'test'%3B - it's basically `any` and, if anything, will break in the future when `Object` is an improved type (E.g. defines statics like `keys`). I'd use  `any` in the future, or specify the index signature manually (E.g. `{ [key: string]: any }`).